examples/golang/topic-creation/topic-creation.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+)
+
+func main() {
+	topic := flag.String("t", "", "The name of the topic")
+	attributes := flag.String("a", "", "Topic attributes needed")
+	attributesmap := map[string]*string{}
+	json.Unmarshal([]byte(*attributes), &attributesmap) // convert JSON string to Go map
```

Comment:
please  check for errors in the `json.Unmarshal()` call.
users may pass in malformed json

Reply:
Thanks for this. I had not tested for this.